Feature branch

kaizipaul opened the pull request:
In this PR:
-Added Linters
-Added HTML and CSS files 
-Added heading to HTML file
-Added styling to the heading using CSS 
-Linked the CSS file to the HTML file
-Updated the ReadMe file and .gitignore file

**Thank you for Reviewing my PR**

kaizipaul committed:
added linters

kaizipaul committed:
added html file

kaizipaul committed:
added styles.css

kaizipaul committed:
corrections to index.html

kaizipaul committed:
corrected styles.css

kaizipaul committed:
corrections to styles.css part 2

kaizipaul committed:
corrections to styles.css part 3

kaizipaul committed:
corrections to styles.css part 4

kaizipaul committed:
here goes nothing :/

levy002 reviewed (changes requested):
## Hi @kaizipaul :wave: , 
### I'm Levy your PR reviewer,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://media.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif)

## Highlights
- Correct usage of git-flow :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

- [x] Kindly make sure to add a good PR title and PR description. It makes your repo looks more professional. :wink: 
   - Your PR
![image](https://user-images.githubusercontent.com/87197412/184990429-220f7e97-6c26-40b9-b3fa-5eac22e89c9c.png)
   - Example of good PR
![image](https://user-images.githubusercontent.com/87197412/184990532-da778ab4-c789-4136-90a9-629eedd969da.png)

- [x] Kindly make sure to write your commit message in imperative tense and capitalized. Please check this [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to understand it clearly.

- [x] Kindly add descriptive README file to your project to make sure your looks repo more proffesional. Please take a step back and check [this template](https://github.com/microverseinc/readme-template) but remember you can always customize it and make more it professional and personal :sunglasses: 

- [x] Let's make sure to add `test.md` in `.gitignore`. `test.md`file will be create locally but will not be pushed to our github repository because it will be already added into our `gitignore` file. Kindly make sure you solve it as requested in the project requirements. :wink: 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

kaizipaul committed:
Update on ReadMe file and corrections to the .gitignore file

kaizipaul commented:
In this PR:
-Added Linters
-Added HTML and CSS file
-Added heading to HTML file
-Added styling to CSS file
-Updated ReadMe and .gitignore file

kaizipaul commented:
In this PR:
-Added Linters
-Added HTML and CSS files 
-Added heading to HTML file
-Added styling to the heading using CSS 
-Linked the CSS file to the HTML file
-Updated the ReadMe file and .gitignore file

**Thank you for Reviewing my PR**

HAMMAS-SALEEM commented on README.md:
```diff
@@ -1 +1,10 @@
-# hello-microverse
\ No newline at end of file
+# Hello Microverse! Project
+>This project is made as a practise for HTML and CSS, and professional code writing.
+
+## Built with 
+- HTML
+- CSS
+
+## Author
+**Paul Kaizirege**
+- Github : [@kaizipaul]
```
- [x] You didn't add all the fields from this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) in your readme file such as **Contributing**, **Show your support**, **Acknowledgement**. This is your readme file so please take it seriously :blush:
- [x] Also, add a license file in your repository you can add using this [template](https://github.com/microverseinc/readme-template/blob/master/MIT.md) :blush:
- [x] Don't forget to add your name, current year, and app type in your MIT.md file after adding it in your project :+1:

HAMMAS-SALEEM commented on .gitignore:
```diff
@@ -0,0 +1,17 @@
+# Ignore Mac system files
+.DS_store
+
```
### Wrong Place for Summary
![summary](https://user-images.githubusercontent.com/93039359/185093371-05ef03ed-b9ac-4955-9792-539235a1a23d.png)

### RIght Place for Summary
![right1](https://user-images.githubusercontent.com/93039359/185093390-e4c2a8fc-0f9f-423c-8792-bd098a511be2.png)

- [x] You should not add your summary in the message. Please follow the steps to update your summary :blush:
- **Step 1**

![step1](https://user-images.githubusercontent.com/93039359/185094224-afdd945d-0ac8-4c9d-8d6a-f542c7007aba.png)
- **Step 2**

![step2](https://user-images.githubusercontent.com/93039359/185094263-c25668c1-e7a7-47a8-87f6-1d0ebb2ef68e.png)

- **Step 3**
![step3 (copy)](https://user-images.githubusercontent.com/93039359/185094283-8c22e596-1b40-4e06-acd1-901911bda96d.png)

HAMMAS-SALEEM reviewed (changes requested):
# First of all Welcome to Microverse :tada: :bouquet:
# Required Changes :recycle:
![almost-there](https://user-images.githubusercontent.com/93039359/178569876-9fcda5e8-6819-4569-95ee-acb393903a05.jpg)

### Hey @kaizipaul :wave:

### Highlights :1st_place_medal:
 
:heavy_check_mark: Meaningful commits
:heavy_check_mark: Added .gitignore
:heavy_check_mark: Following Github Flow

Good job on implementing some of the changes requested by the previous reviewer, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional Suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification. Otherwise reach out to me on slack which is more fast. You can reach out to me on slack by searching my name in **full-time-students** channel :+1:

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

kaizipaul committed:
Corrections to ReadMe and MIT files

sumairq reviewed (approved):
### Hi @kaizipaul  , 

<img src="https://media4.giphy.com/media/BPJmthQ3YRwD6QqcVD/giphy.gif"/>

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
